pints/_optimisers/__init__.py

```diff
@@ -324,6 +329,46 @@ def __init__(
         # Check if minimising or maximising
         self._minimising = not isinstance(function, pints.LogPDF)
 
+        # Transform everything
+        # From this point onward the optimisation will see only the
+        # transformed search space and will know nothing about the model
+        # parameter space.
+        if transform:
+            if self._minimising:
+                function = transform.apply_error_measure(function)
+            else:
+                function = transform.apply_log_pdf(function)
+            x0 = transform.to_search(x0)
+            #
```

Comment:
This should go in a docstring somewhere. Or do we need a method `transform.apply_covariance_matrix` ??

Reply:
Doing `transform.convert_standard_deviation` and `transform.convert_covariance_matrix` now.